google-cloud-storage/lib/google/cloud/storage/service.rb

```diff
@@ -126,19 +126,26 @@ def patch_bucket bucket_name,
                          predefined_default_acl: nil,
                          if_metageneration_match: nil,
                          if_metageneration_not_match: nil,
-                         user_project: nil
+                         user_project: nil,
+                         options: {}
           bucket_gapi ||= Google::Apis::StorageV1::Bucket.new
           bucket_gapi.acl = [] if predefined_acl
           bucket_gapi.default_object_acl = [] if predefined_default_acl
 
+          if options[:retry].nil?
```

Comment:
Could you add docs for the option? Also just wanted to confirm that this will be something you can add across the board to library methods.

Reply:
I update OVERVIEW doc which explains how to use the `options` field overall. Also updated other methods to accept `options`. We've a separate PR raised already for non-idempotent operations.